README.mkdn

```diff
@@ -11,7 +11,7 @@ You can do this by using these commands:
     (Go to repo you are patching, make your changes and commit)
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit